migrations/versions/12780a039923_populate_database.py

```diff
@@ -0,0 +1,82 @@
+"""populate database
+
+Revision ID: 12780a039923
+Revises: 67ac3b7d5c2f
+Create Date: 2021-09-14 14:44:50.802230
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '12780a039923'
+down_revision = '67ac3b7d5c2f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('profile',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=500), nullable=True),
+    sa.Column('first_name', sa.String(), nullable=True),
+    sa.Column('last_name', sa.String(), nullable=True),
+    sa.Column('avatar_url', sa.String(length=500), nullable=True),
+    sa.Column('bio_rendered', sa.String(), nullable=True),
+    sa.Column('interests', sa.String(), nullable=True),
+    sa.Column('before_rc', sa.String(), nullable=True),
+    sa.Column('during_rc', sa.String(), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('stint',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('profile_id', sa.Integer(), nullable=True),
+    sa.Column('type_stint', sa.String(), nullable=True),
+    sa.Column('start_date', sa.Date(), nullable=True),
+    sa.Column('end_date', sa.Date(), nullable=True),
+    sa.Column('title', sa.String(), nullable=True),
+    sa.ForeignKeyConstraint(['profile_id'], ['profile.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.add_column('nicety', sa.Column('stint', sa.Integer(), nullable=True))
+    op.drop_constraint('nicety_author_id_target_id_end_date_key', 'nicety', type_='unique')
+    op.create_unique_constraint(None, 'nicety', ['author_id', 'target_id'])
```

Comment:
This constraint needs to include the stint, so that one person can write two niceties for someone who has attended RC twice.